runtime/moonbeam/src/lib.rs

```diff
@@ -1211,6 +1216,7 @@ construct_runtime! {
 		RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage} = 2,
 		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 3,
 		ParachainInfo: parachain_info::{Pallet, Storage, Config} = 4,
+		RootTesting: pallet_root_testing::{Pallet, Call, Storage} = 5,
```

Comment:
Are we positive index 5 was never used?

Reply:
Good question, I can move it to end of line just to be safe. @notlesh opinions?